Project-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,8 @@
 import React from 'react';
 import './Header.css';
+
+const ImageThumbnail = () => {
+    return <img src= ' https://tk-assets.lambdaschool.com/1c1b7262-cf23-4a9f-90b6-da0d3c74a5c6_lambdacrest.png' alt="" className = "lambdaLogo"/>;
```

Comment:
think about ways that you could pass things like the img src as props. The reason for this being is that currently this component is only reusable if you wanted to use the same image. What if you had multiple users posting and each had to have their own image. This could only be done if we're keeping our props free from data and just defining it later. Pass as props and let the parent component figure it out.